app/components/e-heading.md

```diff
@@ -0,0 +1,51 @@
+### Headings
+
+```
+<div>
+  <e-heading tagName="h1" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
```

Comment:
- [ ] Use `boolean` props without value. This way if they are defined, they will be `true`, if missing `false`. (Also on following lines)